### PR TITLE
OADP 3765 Updated PR Data Mover new parameters

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP 1.3.1

OCP 4.13 +

GA April 16th 2024

Resolves - https://issues.redhat.com/browse/OADP-3765

Preview - 

https://74205--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-backup-restore-csi-snapshots.html#oadp-1-3-backing-csi-snapshots_oadp-backup-restore-csi-snapshots

https://74205--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-backup-restore-csi-snapshots

### CODE REVIEW

**Comment by CarmiWisemon** [comment]
/label peer-review-needed

A similar PR was previously reviewed and merged  https://github.com/openshift/openshift-docs/pull/73847. 

I mistakenly asked for it to be merged before the GA, and then had to ask to revert the merge. 

This PR is instead of that PR that was reviewed, merged and reverted.

Thank you for your help and understanding.

File: backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc [hunk 1]
```diff
@@ -38,11 +38,16 @@ spec:
...
+      defaultSnapshotMoveData: true
+      defaultVolumesToFSBackup: <4>
+      featureFlags:
+      - EnableCSI
 # ...
 ----
 <1> The flag to enable the node agent.
 <2> The type of uploader. The possible values are `restic` or `kopia`. The built-in Data Mover uses Kopia as the default uploader mechanism regardless of the value of the `uploaderType` field.
 <3> The CSI plugin included in the list of default plugins.
+<4> Set to `true` if you use Data Mover only for volumes that opt out of `fs-backup`. Set to `false` if you use Data Mover by default for volumes.
```
Comment (PrasadJoshi12) [inline backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc:50]: DPA flags are introduced in 1.3.1 here you can keep the OADP 1.3.1 and later string but for backup spec it's not needed

**Review by xenolinux — COMMENTED** [review]
One suggestion; LGTM
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

File: backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc [hunk 2]
```diff
@@ -38,11 +38,16 @@ spec:
...
+      defaultSnapshotMoveData: true
+      defaultVolumesToFSBackup: <4>
+      featureFlags:
+      - EnableCSI
 # ...
 ----
 <1> The flag to enable the node agent.
 <2> The type of uploader. The possible values are `restic` or `kopia`. The built-in Data Mover uses Kopia as the default uploader mechanism regardless of the value of the `uploaderType` field.
 <3> The CSI plugin included in the list of default plugins.
+<4> OADP 1.3.1 and later: Set to `true` if you use Data Mover only for volumes that opt out of `fs-backup`. Set to `false` if you use Data Mover by default for volumes.
```
[the remark this answers is not in the document]
Comment (CarmiWisemon) [reply backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc]: @xenolinux 
Thank you for review. I implemented your suggestion

**Pull request merged**